context/app/static/js/components/Maintenance/style.js

```diff
@@ -0,0 +1,16 @@
+import Container from '@material-ui/core/Container';
+import styled from 'styled-components';
+
+const Background = styled.div`
+  background-color: ${(props) => props.theme.palette.warning.main};
+  margin-top: -16px; // Override the Header's margin.
```

Comment:
I'd prefer not to use negative margin here. I'd like to use a separate `MaintenanceHeader`. I also think we can remove the default `margin-bottom` from the Header and incorporate it in a wrapper component for a route which will allow you to specify whether the route's content should be have a width constraint, top margin, or bottom margin.